components/st-microelectronics/stm-api.stanza

```diff
@@ -130,132 +138,177 @@ defn match-pin? (pin-ref:Ref power:String) :
   if p-ref == power :
     true
 
-defn find-pins-list (obj:JITXObject my-pin:String) :
+; return list of pins that match name (exactly) / don't check for empty
+defn get-pin-list (obj:JITXObject pin-id:?) :
```

Comment:
Can you separate function parameters with a comma?
It's perfectly valid without, but it's an unspoken standard.
`(obj:JITXObject, pin-id:?)`

Reply:
Oh, ok will do.